section2/exercises/if-statements.js

```diff
@@ -62,12 +70,22 @@ When numQuarters = 2, program should log "I have enough money for a gumball"
 When numQuarters = 3, program should log "I have enough money for a gumball"
 */
 
-var numQuarters = 0;
-
+// var numQuarters = 0;
+// var numQuarters = 1;
+var numQuarters = 2;
+// var numQuarters = 3;
 // Write your conditional here
 console.log("I have enough money for a gumball");
 console.log("I don't have enough money for a gumball");
-
+if (numQuarters === 0) {
```

Comment:
Not required to redo but I'd encourage you to think about before mod 1 -

Notice that lines 81 and 83 and are the same, so are 85 and 87. Could you (what we call DRY -Don't Repeat Yourself up your code) modify the code so you only need to write those lines of code ONCE? This would mean changing the way you write the conditions.